src/main/java/frc/robot/constants/AprilTagLocalizationConstants.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.constants;
+
+import static edu.wpi.first.units.Units.Meters;
+import static edu.wpi.first.units.Units.Seconds;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.units.measure.Time;
+
+/** Add your docs here. */
+public class AprilTagLocalizationConstants {
+  public static class LimelightDetails {
+    public String name;
+    public Matrix<N3, N1> closeStdDevs;
+    public Matrix<N3, N1> farStdDevs;
+    public Matrix<N3, N1> inverseOffset;
```

Comment:
These all should be at least `final` to assure noone can manipulate these from outside the class.  And should these all be `private` rather than `public`?  What is the use case to why you want these publicly accessible?